src/containers/drawing-tool-controls/component.js

```diff
@@ -0,0 +1,226 @@
+import React, { useCallback, useState, useMemo } from 'react';
+
+import cx from 'classnames';
+import { isEmpty } from 'lodash-es';
+import PropTypes from 'prop-types';
+import Link from 'redux-first-router-link';
+
+import Button from 'components/button';
+import Icon from 'components/icon';
+import Modal from 'components/modal';
+import SearchLocation from 'components/search-location';
+
+import styles from './style.module.scss';
```

Comment:
??

Reply:
Leftovers of the first iteration of the drawing tool. That container is not even being used. Will remove.